pkg/crypto/rsa_test.go

```diff
@@ -0,0 +1,54 @@
+package crypto
+
+import (
+	"io/ioutil"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+const (
+	publicKeyDERFile  string = "tests/public_key.der"
```

Comment:
can you add a comment about each of these test files? So it is explicit how they are generated and what's their purpose (e.g. "broken public key" and so on)

Reply:
okay